weekly-updates/Georgia-Aitkenhead/weekly-planning-sessions.md

```diff
@@ -35,6 +36,14 @@ Notes from Georgia and Bastian's weekly planning sessions
 * [12 December 2022](#date-12-december-2022)
 * [28 November 2022](#date-28-november-2022)
 
+### Date 18 April 2024
```

Comment:
```suggestion
### Date 10April 2024
```

Also the planning for the CW workshop could fit in, depending on how the rest goes!